packages/web/components/templates/homeFeed/HomeFeedContainer.tsx

```diff
@@ -259,7 +259,11 @@ export function HomeFeedContainer(props: HomeFeedContainerProps): JSX.Element {
         const username = viewerData?.me?.profile.username
         if (username) {
           setActiveCardId(item.node.id)
-          if (item.node.state === State.PROCESSING) {
+          if (
```

Comment:
Do we still need this bit with your API change?

Reply:
Actually we didn't make changes to the API. The content `<p>We were unable to parse this page.</p>` is added in the last commit.
We were sending user to `/links/id` page when this item is still processing (for refreshing) but here we need to send user to `/slug` page in order to see the unparsable content.
We can also send the user to `slug` page here and do the refreshing

Reply:
Ahh ok